guides/common/modules/proc_configuring-satellite-for-outgoing-emails.adoc

```diff
@@ -3,6 +3,10 @@
 
 To send email messages from {ProjectServer}, you can use either an SMTP server, or the `sendmail` command.
 
+.Prerequisites
+
+* Some SMTP servers with anti-spam protection or grey-listing features are known to cause problems, to setup outgoing email with such service either install and configure a vanilla SMTP service on {ProjectServer} for relay or use `sendmail` command instead.
```

Comment:
Minor grammar suggestions.

```suggestion
* Some SMTP servers with anti-spam protection or grey-listing features are known to cause problems. To setup outgoing email with such a service either install and configure a vanilla SMTP service on {ProjectServer} for relay or use the `sendmail` command instead.
```

Also, doesn't the sendmail command rely on a local mailserver already?

Reply:
Just need a comma before either :)

Reply:
> doesn't the sendmail command rely on a local mailserver already?

Well it does as far as I know, but the gist of my message is: don't configure against a remote enterprise-configured (tightened) SMTP. Use a local one and let it do the job of the delivery, not Ruby.

Amended, commas - I don't use so no idea where exactly to put them ;-)